.circleci/config.yml

```diff
@@ -10,6 +10,11 @@ jobs:
       - checkout
       - restore_cache:
           key: node-modules-{{ .Branch }}-{{ checksum "ui/package.json" }}
+      - run:
```

Comment:
Can you also manually test with Framingham?

Reply:
Testing 1000 genomes, Framingham, and AMP PD looks good